### PR TITLE
[52] Added user_level table with trigger

### PR DESCRIPTION
**Pull request by Szyntos**
Added user_level table with trigger

### CODE REVIEW

**Comment by AnCichocka** [comment]
@Szyntos would it be difficult to add prev and next level as well?

File: backend/src/main/kotlin/backend/userLevel/UserLevel.kt [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    @JoinColumn(name = "user_id", referencedColumnName = "user_id")
+    var user: Users,
+
+    @ManyToOne
+    @JoinColumn(name = "level_id", referencedColumnName = "level_id")
+    var level: Levels,
+
+    @Column(name = "label", nullable = false, length = 256)
+    var label: String,
+) {
```
Comment (Jullija) [inline backend/src/main/kotlin/backend/userLevel/UserLevel.kt:30]: add edition_id, because it is missing
image

Comment (Jullija) [reply backend/src/main/kotlin/backend/userLevel/UserLevel.kt:30]: Untitled

**Review by Jullija — APPROVED** [review]
imho gut

**Pull request merged**